### DISCUSSION TITLE
Is their a cost for an enterprise to use mkdocs?

### DISCUSSION THREAD

**Discussion by russurquhart** [mkdocs/mkdocs#3006] [Q&A]
Hi,
Is these a cost/restrictions for an enterprise wanting to use MKDocs for their doc website?

Thanks

**Comment by oprypin** [comment]
No, it's free open source software, it doesn't restrict commercial use or even distinguish it in any way.
https://github.com/mkdocs/mkdocs/blob/master/LICENSE
* Even if you want to arbitrarily pay, there's currently no way to do that, see https://github.com/mkdocs/mkdocs/discussions/2969
